packages/server/src/config.ts

```diff
@@ -94,8 +94,8 @@ function mailerConfigFromEnv(env: EnvVariables): MailerConfig {
 	return {
 		enabled: env.MAILER_ENABLED !== '0',
 		host: env.MAILER_HOST || '',
-		port: Number(env.MAILER_PORT || 587),
-		secure: !!Number(env.MAILER_SECURE) || true,
+		port: Number(env.MAILER_PORT || 465),
+		encryption: env.MAILER_SECURE || 'smtps',
```

Comment:
I think we could rename it to MAILER_SECURITY, otherwise it looks like a Boolean. Also as values I didn't see smtps mentioned much anywhere. It's usually either ssl/tls or starttls, so maybe we could use that as values? Maybe just "ssl" and "starttls"? (Or empty string for none)

Reply:
And encryption should be renamed "security" I guess (or "connectionSecurity" but that's a bit verbose)

Reply:
1) Yep, `MAILER_SECURITY` is a better name.

2) Yeah, I think `ssl` is better than `smtps` -- according to [RFC 8314](https://datatracker.ietf.org/doc/html/rfc8314#section-7.3):
>   Historically, port 465 was briefly registered as the "smtps" port.
>   This registration made no sense, as the SMTP transport MX
>   infrastructure has no way to specify a port, so port 25 is always
>   used.  As a result, the registration was revoked and was subsequently
>   reassigned to a different service.

  so technically `smtps` refers to an old usage of port 465, rather than generically SMTP over SSL/TLS.

Reply:
2 things:

- SSL is not used anymore. This protocol is dead. It has been for years. It has been superseded by TLS. SSL is not even comp9iled into most apps anymore.
- while it is true that smtps is not the correct name for port 465, it is still used as such by many admins. there is no STARTTLS on port 465 and thus a TLS handshake has to happen upon connect. If you want to change `smtps`, it should be either changed to `tls` or `submissionOverTls`.

Reply:
@tessus If we want to be completely RFC-compliant, we should probably call it `submissions`. The point about SSL being dead is well-taken -- we should prefer `tls` over `ssl` in any case.

Reply:
`submissions` - yep, this is true. however, it sounds a bit weird, because it looks like a plural, even though it is the singular + s for secure.

But it's better than ssl. I'm ok with either `tls` or `submissions`. Let's Laurent decide what he likes best.

Reply:
We had this discussion on this repo at some point regarding ssl/tls. SSL indeed is deprecated but it's still a term [that's widely used](https://trends.google.co.uk/trends/explore?geo=GB&q=%22ssl%20certificate%22,%22tls%20certificate%22) to refer to both SSL and TLS. Many implementations also refer to "ssl/tls" but because "ssl_tls" looks ugly, I've suggested just "ssl". I'm not keen on just "tls" or things like "submissionOverTls" because I've never seen it anywhere.

If you look at an article like this: https://www.fastmail.help/hc/en-us/articles/360058753834-SSL-TLS-and-STARTTLS You have "TLS, SSL" and "STARTTLS", so we should use a variation of this. I think either "ssl_tls" or just "ssl".

Reply:
IMO this is wrong, but it's your project. The reason why it is still widely used is because people (who should know better) are ok with using the term SSL instead of TLS.

But I will shut up and no longer try to beat a dead horse.

Reply:
It's not really about what's right or wrong, but about what's commonly used. Also I assume Nodemailer still supports SSL and probably some old servers rely on this, so using "ssl" is not entirely incorrect. How about "ssl_tls" then? It just says plainly what it is, since both SSL and TLS are supported, and that would match what seems to be commonly used.

Reply:
Afaik the SSL protocol is nowhere used anymore. Even TLS 1.0 and 1.1 have been forcefully deprecated by banks and payment providers. E.g. Stripe won't work with TLS  < 1.2.

I do not know if nodemailer really supports SSL. As mentioned before SSL has been removed from many code bases. I would have to investigate.

But I believe `ssl_tls` is certainly better than just `ssl`.

Reply:
@kc9jud what's the status of this?

Reply:
@tessus just got back from traveling -- I will try to finish this up this evening or tomorrow.